lib/fluent/plugin/out_influxdb.rb

```diff
@@ -20,9 +21,10 @@ def initialize
 
   def configure(conf)
     super
-    @influxdb = InfluxDB::Client.new @dbname, host: @host, 
-                                              port: @port, 
-                                              username: @user, 
+    @remove_tag_prefix = conf['remove_tag_prefix']
```

Comment:
We have HandleTagNameMixin for this case.
Please check this include https://github.com/fujiwara/fluent-plugin-suppress/blob/0666a2d2c72599b6b2cc27b765f4f095efcbaa0e/lib/fluent/plugin/out_suppress.rb#L4 and this `filter_record` https://github.com/fujiwara/fluent-plugin-suppress/blob/0666a2d2c72599b6b2cc27b765f4f095efcbaa0e/lib/fluent/plugin/out_suppress.rb#L67